plutus-playground-client/src/MainFrame.purs

```diff
@@ -235,21 +233,8 @@ eval (HandleDropEvent event next) = do
   saveBuffer contents
   pure next
 
-eval (HandleMockchainChartMessage EC.Initialized next) = do
-  updateChartsIfPossible
```

Comment:
hmm, are you sure you want to get rid of this?

Reply:
100% sure. In fact I'm proud of it. The echarts wrapper requires it. My chartist wrapper just changes automatically when the data changes. 😎 

(There are situations where this could be a performance problem, of course, nothing we're even approaching. And even if we did, I wouldn't solve it the way they have.)